pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -192,9 +192,11 @@ func extractRawAttributes(log azureLogRecord) map[string]any {
 	}
 	attrs[azureOperationName] = log.OperationName
 	setIf(attrs, azureOperationVersion, log.OperationVersion)
+
 	if log.Properties != nil {
-		attrs[azureProperties] = *log.Properties
+		copyProperties(log.Category, log.Properties, attrs)
```

Comment:
Thinking about these changes some more, they end up being a breaking change regardless of whether or not the semconv have approved the attribute names.  This change moves the data out of the `azure.properties` and into new, semantically relevant, fields.  The result is the user needs to good look for their data in a new place - a better place, but new nevertheless.

Here is my proposal for how to move these breaking changes forward to give users time to adjust and and time for the semconv to discuss the names. Lets add 2 new feature gates that will control how the data is produced.
1. A feature gate that, when enabled, will translate the data in the old way and the new way.
2. A feature gate that, when enabled, will translate the data in only the new way.

If both feature gates are enabled, the first gate will take precedence.

Reply:
I'd suggest that this is a new translator, so it's opt-in in the config for this work to come into effect.

Reply:
A new translator (or a new flow in this azure translator) works too.  The eventhub receiver will need 2 new formats to help with the transition tho, 1 format that emits only the new data, and 1 format that emits the new data and the old data.

Reply:
@atoulme @crobert-1 please weigh in as code owners on how you'd like to see these changes integrated.

Reply:
Also @djaglowski

Reply:
@TylerHelmuth I believe that providing a way to only emit old or new is sufficient. I can see an argument for both; but I do agree this could be considered a breaking change.